### DISCUSSION TITLE
Dark/Light mode switch

### DISCUSSION THREAD

**Discussion by fabrizioschiavi** [fabrizioschiavi/pragmatapro-semiotics#4] [Ideas]
@amake what do you think to offer the possibility to switch from Dark to Light mode through a button in index.html?

**Comment by amake** [comment]
I added this in 3eb90785fcef47b799a1da7775ceb1128a3e12e8. Let me know what you think.

(You can see how much complexity it adds!)

**Reply by fabrizioschiavi** [reply]
Great job!!!
